### ISSUE TITLE
Loading the PBA from a rescue image -- (was Quick question)

### ISSUE THREAD

**Issue by DrEmpiricism** [Drive-Trust-Alliance/sedutil#131]
If one were to go about setting sedutil directly from the rescue USB, what is the syntax to direct the loadPBAImage command to the PBA.img located on a drive?  Any recommendations where to place the PBA?  Obviously I can't load it to the rescue disk since Windows does not recognize its format.  I also can't mount the USB since it's just a mini-Linux core.

I ask because on this MX300 I'm toying around with, I am unable to run the sedutil commands within Windows.  For some reason it just locks the entire machine after a few seconds and a cold-boot is required.  None of my other SEDs do that, but the MX300 does it on every PC I've tried it on.

That said, I am able to do the initialsetup, take ownership, change passwords, etc. but if I do not load a PBA, the PC just boots to an all black screen until I turn locking off, so I'd like to use the rescue USB to also load the PBA but am not well versed in Linux at all and Google isn't giving me exactly what I need since this is sort of an odd question.

Thanks.

**Comment by Lutwick** [comment]
The sedutil console commands in the rescue USB stick's Linux are the same as under Windows. The only difference might be that you just write "sedutil-cli" instead of "sedutil-cli.exe".

PS: Please try to use more descriptive topics for your requests than "Quick question". ;-)

**Comment by DrEmpiricism** [comment]
They are most definitely not the same.

**Comment by Lutwick** [comment]
Can you give an example of where they are not the same?

Of course, your disk name is different under Linux than under Windows.
In Windows it is something like \\.\PhysicalDrive0,
whereas in Linux it is something like /dev/sda.

sedutil.exe --scan    in Windows is
sedutil --scan    in Linux.

From there you get your disk names.

**Comment by DrEmpiricism** [comment]
There are far more differences between Linux commands and Windows commands, albeit the most common sedutil-cli commands, like you posted, are the same with the exception of drive location.

However that was not my question, though my question needs not be answered now since it's long been long rectified after remembering I had to mount a device manually for Linux to see it (I have not used Linux in quite a while, clearly).

For simplicity's sake, I added the PBAs to the rescue image itself and created a UEFI version, too.  It's just significantly quicker to boot into the USB prior to a Windows installation and being able to do everything from a small live environment.
